src/PrestaShopBundle/Entity/ApiAccess.php

```diff
@@ -124,53 +137,53 @@ public function setClientId(string $clientId): ApiAccessInterface
     /**
      * {@inheritdoc}
      */
-    public function getClientSecret(): string
+    public function getClientName(): string
     {
-        return $this->clientSecret;
+        return $this->clientName;
     }
 
     /**
      * {@inheritdoc}
      */
-    public function setClientSecret($clientSecret): ApiAccessInterface
+    public function setClientName(string $clientName): ApiAccessInterface
```

Comment:
```suggestion
    public function setClientName(string $clientName): self
```
Can you please remove the usage of this interface? It's supposed to be removed, and using`self` for fulent setters is more flexible than using hard-coded types

Reply:
You also want me to completly remove the interface ?